src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -209,6 +210,7 @@ TransportResponse handleOpenSearchRequest(OpenSearchRequest request) throws Exce
     IndicesModuleResponse handleIndicesModuleRequest(IndicesModuleRequest indicesModuleRequest, TransportService transportService) {
         logger.info("Registering Indices Module Request received from OpenSearch");
         IndicesModuleResponse indicesModuleResponse = new IndicesModuleResponse(true, true, true);
+        sendActionListenerOnFailureRequest(transportService, new Exception("Test failure"));
```

Comment:
Per comments on the companion PR, really try to avoid throwing broad Exceptions.  

Also I don't understand why we are throwing a "test" failure in the main code.  This even looks like it's in a non-failure path.